.github/workflows/ci.yml

```
@@ -0,0 +1,49 @@
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  workflow_dispatch:
+
+jobs:
+  test:
+    name: Test, Lint and Build
+    runs-on: ubuntu-latest
+    
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)